.github/workflows/tests.yml

```diff
@@ -499,7 +499,7 @@ jobs:
 
       - run: |
           set -o pipefail
-          POSTGRES=${{ (matrix.database == 'Postgres') && 1 || '' }} WORKERS=${{ (matrix.arrangement == 'workers') && 1 || '' }} COMPLEMENT_DIR=`pwd`/complement synapse/scripts-dev/complement.sh -json 2>&1 | synapse/.ci/scripts/gotestfmt
+          POSTGRES=${{ (matrix.database == 'Postgres') && 1 || '' }} WORKERS=${{ (matrix.arrangement == 'workers') && 1 || '' }} COMPLEMENT_DIR=`pwd`/complement COMPLEMENT_ALWAYS_PRINT_SERVER_LOGS=1 synapse/scripts-dev/complement.sh -json -run TestFed 2>&1 | synapse/.ci/scripts/gotestfmt
```

Comment:
(does `TestFed` need to be `'TestFed.*'`---IIRC -run takes a regex rather than a substring?)

Reply:
It does take a regex but it's not anchored to be a full match, so substring works as well